### ISSUE TITLE
Code never run

### ISSUE THREAD

**Issue by chesstrian** [indutny/sticky-session#33]
This can be a simple question, but why the message `server started on 3000 port` on code taken from example, never is shown?.

```js
  // Worker code
  server.once('listening', function() {
    console.log('server started on 3000 port');
  });
```

**Comment by indutny** [comment]
Because the condition should be reversed! Sorry!

**Comment by chesstrian** [comment]
No matter where the code is put, never is called. Still the library works without this, but just wanted to know if there is a way to make it works.

**Comment by indutny** [comment]
Ouch, fixed! Please update to 1.0.1

**Comment by chesstrian** [comment]
Thanks @indutny, I implemented a simple example https://github.com/chesstrian/sticky-session-express.
